students/viveksyngh/quiz.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"encoding/csv"
+	"os"
+	"fmt"
+	"strings"
+	"log"
+	"flag"
+	"time"
+	"math/rand"
+) 
+
+type Question struct {
+	question string
+	answer string
+}
+
+func getQuestions(filePath string) ([]Question) {
+	file, err := os.Open(filePath)
+	if(err != nil){
+		log.Fatal("Failed to open file.")
+	}
+	reader := csv.NewReader(file)
+	questionList, err := reader.ReadAll()
+	if(err != nil) {
+		log.Fatal("Failed to parse CSV file.")
+	}
+	questions := make([]Question, 0)
+	
+	for _, question := range questionList {
+			questions = append(questions, Question{strings.TrimSpace(question[0]), 
+													strings.TrimSpace(question[1])})
+	}
+	return questions
+}
+
+func Quiz(questions []Question, timer *time.Timer) (score int){
```

Comment:
There is no need for named return, it's another allocation. If you simply name your function `score` it informs the reader what they expect.